File: Homework3/Question2/src/CoqAuVin.java
public class CoqAuVin extends Chicken {
    public CoqAuVin() {
        super("brown", 0);
    }

    public static void testCoqAuVin() {
        CoqAuVin coq = new CoqAuVin();
        //Test coqauvin
        System.out.println(coq.getColor() == "brown");
        System.out.println(coq.getNumberOfFeathers() == 0);
        System.out.println(coq.isEdible() == true);
    }
}

File: Homework3/CoqAuVin.kt
class CoqAuVin : Chicken {
    constructor() : super("brown", 0)

    companion object {
        fun testCoqAuVin() {
            val coqAuVin = CoqAuVin()
            println(coqAuVin.getColor() == "brown")
            println(coqAuVin.getNumberOfFeathers() == 0)
            println(coqAuVin.isEdible() == true)
        }
    }
}